docs/knowledgebase/runtime/macros.md

```diff
@@ -49,7 +49,7 @@ _Doc. and Example_ to see how macro code is expanded.
 
 **When to Use**
 
-To define storage items in a runtime pallete. A storage item definition includes:
+To define storage items in a runtime palette. A storage item definition includes:
```

Comment:
```suggestion
To define storage items in a pallet. A storage item definition includes:
```

Reply:
btw, is there any situation to use `FRAME pallet` better than `runtime pallet`? Or they could be used interchangeably most of the time?

Reply:
This was part of a **long long and intense** discussion. I still find it okay to say runtime palette but the consensus was to **stay with FRAME palette**. A palette is always part of the runtime, so `runtime palette` is redundant